GOVERNANCE.md

```diff
@@ -175,39 +195,108 @@ The project Steward is currently: **@FredKSchott**
+- Mentoring and guiding other community contributors.
 
 #### Nomination
 
-There is no nomination process for this role. The project steward is responsible for granting and revoking the `@staff` role.
+- To be nominated, a nominee is expected to already be active in technical discussions and performing some of the responsibilities of a TSC member.
+- You can be nominated by any existing Core member (L3 or above). Note: This includes all existing TSC members as well.
+- Once nominated, there will be a vote by existing Core members.
+- See [vote rules & requirements](#voting) for info on how this vote works.
+
+
+### Staff
+
+**Staff** is a special designation for employees of [The Astro Technology Company](https://astro.build/company). 
+
+#### Privileges
 
-#### Leaving Staff
+- `@staff` role on [Discord](https://astro.build/chat)
+- New name color on Discord: **yellow**.
+- Invitation to some private channels on Discord, at the discretion of the project Steward.
+- Invitation to the `staff` team on GitHub.
 
-When someone leaves the Astro Technology Company, they lose staff privileges and return to their original membership level in our governance structure (whatever level they were at before joining staff).
+Staff membership does not grant any additional abilities when it comes to voting and project governance. A Staff member is still eligible for other roles in the community and may still vote as defined by their other roles. For example, a Staff member who is also a part of `@core` will be able to vote as any other `@core` member would.
 
-If that person wishes to continue working on Astro after leaving, they may request a nomination to become an official L2 or L3 contributor. This nomination would follow the normal voting rules & procedure for that role (see [Voting](#voting) below).
+### Alumni
+
+**Alumni** is a special designation for Maintainers (L2 and above) who have stepped away from the project and no longer contribute regularly. See [Retiring a Role](#retiring-a-role-alumni) below for more information.
+
+#### Privileges
+
+- `@alumni` role on [Discord](https://astro.build/chat)
+- New name color on Discord: **light blue**.
+- Invitation to the private `#alumni` channel on Discord.
+
+
+## Retiring a Role (Alumni)
+
+Contributor roles are granted for as long as the person wishes to engage with the project. However, over time an active community member may choose to step away from the Astro project to work on other things. Moving on from a project is a natural and well-understood part of any open source community, and we celebrate it!
+
+**Alumni** is a special designation and role for any person who was once an active maintainer (L2 or above) but is now no longer actively involved. By retiring and joining Alumni you trade-in your current set of roles, privileges, and responsibilities for a new, special Alumni role (which comes with its own set of Privileges, as described above).
```

Comment:
What is the role / privilege of being an Alumni? I imagine it's a way to communicate that you have participated in helping before, but that you're taking a break for some amount of time and aren't involved enough?

Reply:
That's correct! No additional privileges other than what is outlined here.